test/matrix_generator.cc

```diff
@@ -1478,8 +1485,9 @@ void generate_matrix(
                     for (int64_t j = 0; j < nt; ++j) {
                         const int64_t nb = A.tileNb(j);
                         if (A.tileIsLocal(i, j)) {
-                            #pragma omp task firstprivate(i, j, mb, nb, \
-                                                          i_global, j_global)
+                            #pragma omp task slate_omp_default_none shared(A) \
+                                firstprivate(i, j, mb, nb, i_global, j_global, \
+                                inner_const, outer_const)
```

Comment:
Style guide indents 2nd line of firstprivate to align under the `i`. Cf. src/internal/internal_getrf.cc
```
            #pragma omp parallel for num_threads(thread_size) slate_omp_default_none \
                shared(thread_barrier, max_value, max_index, max_offset) \
                shared(top_block, aux_pivot, tiles, bcast_comm) \
                firstprivate( tile_indices, bcast_root, bcast_rank, ib, \
                              diag_len, thread_size, pivot_threshold )
```

Reply:
Updated indent and whitespace.